File: 백준/Silver/11008. 복붙의 달인/복붙의 달인.py
import sys

t = int(sys.stdin.readline())

for i in range(t):
    s, p = map(str, sys.stdin.readline().split())
    cnt = 0

    if p in s:
        cnt = s.count(p)
        s = s.replace(p, "")
        
    sys.stdout.write(str(cnt + len(s)) + "\n")

File: 백준/Silver/11008. 복붙의 달인/복붙의 달인.cc
#include <iostream>
#include <string>
using namespace std;

int main() {
    int t;
    string s, p;
    cin >> t;

    for (int i = 0; i < t; i++) {
        cin >> s >> p;
        int cnt = 0;

        while (s.find(p) != string::npos) {
            s.replace(s.find(p), p.length(), "");
            cnt++;
        }

        cout << cnt + s.length() << endl;
        
    }
    return 0;
}